canonical-web-and-design/anbox-cloud.io
# Replace leadCapture/save2 with the marketo endpoint

## Done
- Post forms to https://ubuntu.com/marketo/submit
- Decapitalise input names
- Update hidden inputs
- Remove recaptcha

## QA

1. Go to https://anbox-cloud-io-221.demos.haus/contact-us
2. Fill in the form and submit it
3. Check it's submitted and redirect to a thank you page on anbox-cloud.io
4. Check the submitted data is on the list in marketo (form ID is 3498)

## Issue / Card

Fixes [#4738](https://github.com/canonical-web-and-design/web-squad/issues/4738)

## Screenshots

[if relevant, include a screenshot]
